lib/proteus/kit.rb

```diff
@@ -50,7 +73,8 @@ def deploy
 
     desc "version", "Show Proteus version"
     def version
-      say "Proteus #{Proteus::VERSION}"
+      version = Proteus::VERSION
+      puts "Proteus #{version}"
```

Comment:
I would avoid having a variable with the same name as the method. Maybe name the variable `version_number`?